frontend/src/components/Header/index.tsx

```diff
@@ -1,14 +1,23 @@
 import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { IStore } from '../../store';
+import { IState } from '../CreateSeason';
+import { toggleSideBar } from '../SideBar/action';
 import './header.css';
 import { Presentational } from './Presentational';
 
-const mapStateToProps = (state: any) => ({
+const mapStateToProps = (state: IStore) => ({
+  isMobile: state.sideBar.isMobile,
```

Comment:
Could this perhaps be a util instead of in store? Since I guess whether or not you're using a mobile doesn't change so much!

Reply:
Good solution! Probably was not wise to do this in the middle of the night.